Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:latest
```

Comment:
Does the final image include go tools/compilers as well?

I wonder if this docker image can be reduced with the multistage build like described here https://medium.com/swlh/reducing-container-image-size-esp-for-go-applications-db7658e9063a

In fact pacoloco most likely will work with `scratch` images if you go this route.

Reply:
This was an oversight on my part, I've changed it to use the scratch base for the resulting image. The resulting image is only 5 megabytes now.